vividus-plugin-azure-service-bus/src/main/java/org/vividus/azure/servicebus/ServiceBusSteps.java

```diff
@@ -0,0 +1,21 @@
+package org.vividus.azure.servicebus;
+
+import org.jbehave.core.annotations.When;
+import org.vividus.azure.servicebus.model.ServiceBusType;
+import org.vividus.azure.servicebus.service.ServiceBusService;
+
+public class ServiceBusSteps
+{
+    private ServiceBusService serviceBusService;
+
+    public ServiceBusSteps(ServiceBusService serviceBusService)
+    {
+        this.serviceBusService = serviceBusService;
+    }
+
+    @When("I send message to service bus $type with name `$name` and payload:$payload")
+    public void sendMessageToServiceBus(ServiceBusType type, String name, String payload)
+    {
+        serviceBusService.send(type, name, payload);
```

Comment:
with current approach there is no much reason to have a dedicated `ServiceBusService` class